src/audio_core/audio_renderer.cpp

```diff
@@ -315,9 +318,18 @@ void AudioRenderer::QueueMixedBuffer(Buffer::Tag tag) {
 }
 
 void AudioRenderer::ReleaseAndQueueBuffers() {
+    s16 thread_counter = 0;
     const auto released_buffers{audio_out->GetTagsAndReleaseBuffers(stream)};
+    queue_mixed_multithread.resize(released_buffers.size());
     for (const auto& tag : released_buffers) {
-        QueueMixedBuffer(tag);
+        queue_mixed_multithread[thread_counter] = std::async(std::launch::async, [=, voice_context = voice_context, 
+          splitter_context = splitter_context, mix_context = mix_context] { 
+            QueueMixedBuffer(tag, released_buffers.size(), thread_counter+1);
+        });
+        thread_counter++;
+    }
+    for (std::size_t thread_to_refrence = 0; thread_to_refrence < released_buffers.size(); thread_to_refrence++) {
```

Comment:
```suggestion
    for (std::size_t thread = 0; thread < released_buffers.size(); thread++) {
```

would make this a tiny bit nicer to grok

Reply:
I would also need to change values inside the loop*